### ISSUE TITLE
Problem installing via virtualenv

### ISSUE THREAD

**Issue by zadoev** [svartalf/python-wargaming#1]
1. Installing without virtualenv

*_:$ sudo pip install wargaming
[sudo] password for *_:
Downloading/unpacking wargaming
  Downloading wargaming-0.1.0-py2.py3-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): six in /usr/local/lib/python2.7/dist-packages (from wargaming)
Requirement already satisfied (use --upgrade to upgrade): requests>=1.0.0 in /usr/local/lib/python2.7/dist-packages (from wargaming)
Installing collected packages: wargaming
Successfully installed wargaming
Cleaning up...

Intalling via virtualenv:

**:$ virtualenv test
New python executable in test/bin/python
Please make sure you remove any previous custom paths from your /home/**/.pydistutils.cfg file.
Installing Setuptools..............................................................................................................................................................................................................................done.
Installing Pip.....................................................................................................................................................................................................................................................................................................................................done.
1. activate virtualenv

**:$ . test/bin/activate
(test)Вторник 2015-03-10 18:03   *_:~/data/tmp
*_:$ pip install wargaming
Downloading/unpacking wargaming
  Downloading wargaming-0.1.0.tar.gz
  Running setup.py egg_info for package wargaming
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/home/**/data/tmp/test/build/wargaming/setup.py", line 6, in <module>
        from wargaming import get_version
      File "wargaming/**init**.py", line 1, in <module>
        from wargaming.games.wot import API as WoT
      File "wargaming/games/wot/**init**.py", line 3, in <module>
        import six
    ImportError: No module named six
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/home/**/data/tmp/test/build/wargaming/setup.py", line 6, in <module>

```
from wargaming import get_version
```

  File "wargaming/**init**.py", line 1, in <module>

```
from wargaming.games.wot import API as WoT
```

  File "wargaming/games/wot/**init**.py", line 3, in <module>

```
import six                                                                                                                                                                                                       
```

ImportError: No module named six                                                                                                                                                                                     

---

Cleaning up...  
Command python setup.py egg_info failed with error code 1 in /home/**/data/tmp/test/build/wargaming  
Storing complete log in /tmp/tmpHxwfEQ                             

*_:$ pip install six  //it's ok
same for requests
*_:$ pip install requests //it's ok

**:$ pip install wargaming
Downloading/unpacking wargaming
  Downloading wargaming-0.1.0.tar.gz
  Running setup.py egg_info for package wargaming
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/home/**/data/tmp/test/build/wargaming/setup.py", line 21, in <module>
        install_requires = open('requirements.txt').read().splitlines()
    IOError: [Errno 2] No such file or directory: 'requirements.txt'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/home/**/data/tmp/test/build/wargaming/setup.py", line 21, in <module>

```
install_requires = open('requirements.txt').read().splitlines()
```

IOError: [Errno 2] No such file or directory: 'requirements.txt'

---

Cleaning up...
Command python setup.py egg_info failed with error code 1 in /home/**/data/tmp/test/build/wargaming
Storing complete log in /tmp/tmpj3r4UV

**Comment by svartalf** [comment]
Fixed in 93fe48e3cf5af697f0fe0693d2da591a7abbdbc4

New version 0.1.1 is uploaded onto PyPI, try to install it.

**Comment by zadoev** [comment]
Fixed, thanks.
